cookbooks/calico/templates/default/compute/bird6.conf.erb

```diff
@@ -44,13 +44,13 @@ protocol direct {
 protocol bgp <%= n[:fqdn].gsub(/[^0-9a-z]/i, '') %> {
```

Comment:
It's not great to put more than a couple lines of ruby in a template. I worked with them a lot in a web app, and they got messy real fast. That said, I'm not sure which way is better here.